projects/gltf_basic_materials/GltfBasicMaterials.cpp

```diff
@@ -331,3 +373,39 @@ void GltfBasicMaterialsApp::InitKnobs()
     GetKnobManager().InitKnob(&mSceneAssetKnob, "gltf-scene-asset", "scene_renderer/scenes/tests/gltf_test_basic_materials.glb");
     mSceneAssetKnob->SetFlagDescription("GLTF asset to load and render");
 }
+
+void GltfBasicMaterialsApp::MouseMove(int32_t x, int32_t y, int32_t dx, int32_t dy, uint32_t buttons)
+{
+    if (!mDefaultCamera) {
```

Comment:
Does this mean you can only move when the scene doesn't have a camera? What happens if the scene does have one?